styles/style.css

```diff
@@ -407,7 +407,7 @@ html {
     color: var(--primary_2);
   }
 
-  .desktop-menu .OS-campaign {
+  .desktop-menu .DV-campaign {
     color: var(--primary_2);
     border: solid 3px var(--primary_2);
     padding: 7px;
```

Comment:
- [ ] From the project requirements, you should stick to [Cindy's design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), But as you can see, the style (width, height, margins and font-sizes) of elements in the list of `Event Speakers` section does not match [Cindy's design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015). Also your images are not showing so make sure that you fix them :pray: 

### Your Design
![image](https://user-images.githubusercontent.com/92303891/227342333-f95d139e-9efd-4bb8-88bd-1beaa037ad7d.png)

### Cindy's Design
![image](https://user-images.githubusercontent.com/92303891/227342478-24b67493-bccd-49ca-96b8-5d82288fd33e.png)

- [ ] **Please on mobile screen you have to list only two speakers as it is in the design :point_down: but your is not so make sure that you work on that** :pray: 
  
  ![image](https://user-images.githubusercontent.com/92303891/227343713-e43802a4-6f14-4464-b2cd-33c3f90c1d40.png)

- [ ] **Kindly note that the style (height) of the containers in the main program section on mobile screen of your project does not match Cindy's design :point_down:** 
  
  ![image](https://user-images.githubusercontent.com/92303891/227345238-bc867762-5c80-4ff6-b353-0881ea829411.png)

- [ ] Please make sure the you have some gap in between the ` Dataverse` and the `2021 Dataverse ...` to match Cindys design
  ### Your design
  ![Screenshot from 2023-03-23 20-29-23](https://user-images.githubusercontent.com/92303891/227348034-19a01cf0-e5ca-4085-9148-34f69698db07.png)

  ### Cindy design
  ![image](https://user-images.githubusercontent.com/92303891/227348591-bf9ee230-8cba-4f68-802c-0fe84b2472b5.png)